packages/@react-aria/listbox/src/useListBox.ts

```diff
@@ -49,7 +49,7 @@ interface AriaListBoxOptions<T> extends AriaListBoxProps<T> {
  * @param props - Props for the listbox.
  * @param state - State for the listbox, as returned by `useListState`.
  */
-export function useListBox<T>(props: AriaListBoxOptions<T>, state: ListState<T>, ref: RefObject<HTMLElement>): ListBoxAria {
+export function useListBox<T>(props: Omit<AriaListBoxOptions<T>, 'children'>, state: ListState<T>, ref: RefObject<HTMLElement>): ListBoxAria {
```

Comment:
My poor documentation generator. It seems to still be showing children. Maybe move the omit to the extends clause on the interface above and see if that works?